vehicle/mkv/README.md

```diff
@@ -0,0 +1 @@
+# MKV
```

Comment:
have we given thought to managing multiple vehicles with different branches (like `MKV/production` and `MKV/development`), instead of new directories? both allow us to keep track of old code without losing it, but with multiple branches we can keep the repo smaller, and when starting a new year, we can just checkout a new branch from the previous year instead of copy and pasting the whole vehicle folder.

Reply:
@wsh32 thoughts? I think there are a few issues: people won't necessarily think to check branches, and it's a bit more difficult to access things across branches than it is to look in a different folder. 

 I don't think we'll need a MKV/prod and MKV/dev branch, I don't see the added benefit. Also I don't think repo size is going to be an issue.

Reply:
This is definitely a complicated issue. We spent a bunch of time talking about it earlier in the week. Especially for the future where we might want to carry a board over from one car to another, it doesn't make sense to duplicate the code. Here's an idea though...

```
.
└── vehicle/
    ├── hardware/
    │   └── example_board/
    ├── model/
    │   ├── mkv/
    │   │   └── BUILD
    │   └── mkvi/
    └── software/
```

Where the `BUILD` file has something like

```python
vehicle(
    name = "mkv",
    boards = [
        "//vehicle/hardware/example_board",
        ...,
    ],
)
```

So we define a `board` in Bazel, which pulls in a `board`, which pulls in a `cc_binary` which is the application that runs on the board. Thoughts?

Reply:
Good points. I'm still not really sure what the right path here is, I think there are pros and cons to both paths, but I think at this point whatever we do will work (and be better than what we have now). Happy to use the initial structure you proposed.

As for the whole BUILD/Bazel think, that doesn't make a whole lot of sense to me. I honestly don't care (I don't feel a strong personal need to learn Bazel) as long as the interface I interact with is simple.